pages/osx/bless.md

```diff
@@ -0,0 +1,19 @@
+# bless
+
+> Set volume bootability and startup disk options.
+
+- FOLDER MODE, To bless a volume with only Mac OS X or Darwin, and create the BootX and boot.efi files as needed:
+
+`bless --folder {{/Volumes/Mac OS X/System/Library/CoreServices}} --bootinfo --bootefi`
+
+- MOUNT MODE, To set a volume containing either Mac OS 9 and Mac OS X to be the active volume:
+
+`bless --mount {{/Volumes/Mac OS}} --setBoot`
+
+- NETBOOT MODE, To set the system to NetBoot and broadcast for an available server:
+
+`bless --netboot --server {{bsdp://255.255.255.255}}`
+
+- INFO MODE, To gather information about the currently selected volume (as determined by the firmware), suitable for piping to a program capable of parsing Property Lists:
```

Comment:
Perhaps this will read better?
```suggestion
- INFO MODE: Gather information about the currently selected volume (as determined by the firmware) - suitable for parsing by scripts:
```